app/src/main/java/dev/ricsarabia/cryptochallenge/ui/detail/OrdersAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package dev.ricsarabia.cryptochallenge.ui.detail
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import dev.ricsarabia.cryptochallenge.databinding.OrderItemBinding
+import dev.ricsarabia.cryptochallenge.domain.BookOrder
+
+class OrdersAdapter: RecyclerView.Adapter<OrdersViewHolder>() {
```

Comment:
It would be better to use ListAdapter instead Recycler View to improve the performance, you can check this link :D 
https://developer.android.com/reference/androidx/recyclerview/widget/ListAdapter